src/include/catalog/postgres/pg_proc.h

```diff
@@ -73,10 +73,15 @@ constexpr proc_oid_t TAN_PRO_OID = proc_oid_t(90);
 constexpr proc_oid_t COT_PRO_OID = proc_oid_t(91);
 constexpr proc_oid_t LOWER_PRO_OID = proc_oid_t(92);
 constexpr proc_oid_t UPPER_PRO_OID = proc_oid_t(93);
+constexpr proc_oid_t CEIL_PRO_OID = proc_oid_t(107);
+constexpr proc_oid_t FLOOR_PRO_OID = proc_oid_t(132);
+constexpr proc_oid_t LOG10_PRO_OID = proc_oid_t(134);
 constexpr proc_oid_t COSH_PRO_OID = proc_oid_t(127);
 constexpr proc_oid_t SINH_PRO_OID = proc_oid_t(128);
 constexpr proc_oid_t TANH_PRO_OID = proc_oid_t(129);
+constexpr proc_oid_t LOG2_PRO_OID = proc_oid_t(133);
 constexpr proc_oid_t VERSION_PRO_OID = proc_oid_t(150);
+constexpr proc_oid_t TRUNCATE_PRO_OID = proc_oid_t(125);
```

Comment:
AFAIK, the oid values don't mean anything. We should just make them increase by one.

Reply:
Fixed